docs/current_docs/ci/quickstart/publish.mdx

```diff
@@ -52,16 +52,25 @@ Just as you can call Dagger Functions individually from the CLI, you can also ca
 - publish the container image to a registry and return the image identifier.
 
 :::tip COMBINING DAGGER FUNCTIONS
-There are various reasons why you might want to write Dagger Functions that call other Dagger Functions. A common reason is that when developing locally, it's quicker and easier to trigger your pipeline using a single command (`dagger call publish ...`) instead of multiple commands (`dagger call test ... && dagger call build ... &&  ...`).
+There are various reasons why you might want to write Dagger Functions that call other Dagger Functions. A common reason is that when developing locally, it's quicker and easier to trigger your pipeline using a single command (`dagger shell -c publish ...`) instead of multiple commands (`dagger shell -c test ... && dagger shell -c build ... &&  ...`).
```

Comment:
```suggestion
There are various reasons why you might want to write Dagger Functions that call other Dagger Functions. A common reason is that when developing locally, it's quicker and easier to trigger your pipeline using a single command (`dagger -c publish ...`) instead of multiple commands (`dagger -c test ... && dagger-c build ... &&  ...`).
```